app/alerts.py

```diff
@@ -67,11 +71,20 @@ def build_departments_geojson():
 
 # ----------------------------------------------------------------------------------------------------------------------
 # Sites markers
+
+# We import the cameras's positions from the API that locates the cameras
+
+# Fetching the response in a variable
+response = api_client.get_sites()
+
+# Getting the json data out of the response
+camera_positions = response.json()
+
 # The following block is used to fetch and display on the map the positions of detection units.
```

Comment:
Do you think you could move this line of comment back up, at line 74 right under the "Sites markers" comment?

This would ensure consistency in the way code sections are presented throughout the file 😊

Reply:
Yes, of course. It's done 😊